diff --git a/netbox/netbox/settings.py b/netbox/netbox/settings.py
index d5a7bfaec..90f05faae 100644
--- a/netbox/netbox/settings.py
+++ b/netbox/netbox/settings.py
@@ -383,7 +383,7 @@ USE_X_FORWARDED_HOST = True
 X_FRAME_OPTIONS = 'SAMEORIGIN'
 
 # Static files (CSS, JavaScript, Images)
-STATIC_ROOT = BASE_DIR + '/static'
+STATIC_ROOT = getattr(configuration, 'STATIC_ROOT', os.path.join(BASE_DIR, 'static')).rstrip('/')
 STATIC_URL = f'/{BASE_PATH}static/'
 STATICFILES_DIRS = (
     os.path.join(BASE_DIR, 'project-static', 'dist'),
